cmd/kusk/internal/overlays/overlay.yaml

```diff
@@ -0,0 +1,12 @@
+overlays: 1.0.0
```

Comment:
Does this file belong here? It looks more like an example

Reply:
using it for the unit test 
```
	if _, err := applyOverlay("overlay.yaml", ""); err != nil {
```